examples/plot_simulate_gamma.py

```diff
@@ -108,6 +108,21 @@
 from hnn_core.viz import plot_psd
 plot_psd(dpls[trial_idx], fmin=20., fmax=100., tmin=tmin)
 
+###############################################################################
+# Finally, we demonstrate the mechanistic link between PING and the GABAA decay
+# time constant (`tau2`). Using the same network/drive configuration as before,
+# we decrease `tau2` from 5 to 2 ms. This will shorten the effective
+# refactory period between L5 pyramidal cell spikes and increase the PING
+# frequency from ~50 to ~65 Hz.
+net.cell_types['L5_pyramidal'].p_syn['gabaa']['tau2'] = 2
+dpls = simulate_dipole(net, n_trials=1)
```

Comment:
so this makes this example too long for CircleCI. Do you want to try adding '-v' to SPHINXOPTS and see if that fixes it:

https://github.com/jonescompneurolab/hnn-core/blob/master/doc/Makefile#L5

other option is to increase the [CircleCI timeout](https://circleci.com/docs/2.0/configuration-reference/#run). In general, we should think how to do continuous integration when the examples become longer and longer. 

MNE has the approach that it runs only the affected example. We could implement something more sophisticated like that if this starts turning out to be the bottleneck.